.gitlab-ci.yml

```diff
@@ -91,6 +98,126 @@ xliff-lint:
     - xmllint --schema ./xliff-core-1.2-strict.xsd
       --noout $(find Resources -name '*.xlf')
 
+unit-tests-php7.2-t3-v10:
+  image: php:7.2
+  stage: test
+  needs:
+    - build-composer-dependencies
+  script:
+    - composer ci:tests:unit
+
+unit-tests-php7.3-t3-v10:
+  image: php:7.3
+  stage: test
+  needs:
+    - build-composer-dependencies
+  script:
+    - composer ci:tests:unit
+
+unit-tests-php7.4-t3-v10:
+  image: php:7.4
+  stage: test
+  needs:
+    - build-composer-dependencies
+  script:
+    - composer ci:tests:unit
+
+functional-tests-php7.2-t3-v10:
+  image: php:7.2
+  services:
+    - mariadb:10
+  stage: test
+  needs:
```

Comment:
Like the unit tests, all functional tests should also depend on linting with the corresponding PHP version.